HACKING.org

```diff
@@ -23,16 +23,26 @@ Git is used to manage the script's state:
 - If a supported OCaml version does not have CI support, run test suite locally.
 
 - In =CHANGES.md=, change =(unreleased)= with the current version (=a.b.c=)
-  and date. Commit the result.
+  and date. In =README.md=, update the recommended version to use in
+  =.ocamlformat= files. Commit the changes.
 
 - Tag
   =dune-release tag=
 
 - Check that release binary has a release version number:
   =dune build @install && dune install --prefix _install && ./_install/bin/ocamlformat --version && rm -rf _install= should output just =a.b.c=
 
+- Linting
+  =dune-release lint=
+
+- Amend the release commit if necessary.
```

Comment:
And update the tag, that's very important otherwise the tag will point to the previous commit. `dune-release tag -f`

Reply:
done